Fancyauth.Model/User.cs

```diff
@@ -2,36 +2,37 @@
 using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 using System.ComponentModel;
+using Fancyauth.Model.UserAttribute;
 
 namespace Fancyauth.Model
 {
     public class User
```

Comment:
(maybe later) TODO: Add methods `IsPersistentGuest` and `IsMember` and `IsGuest` (which are all mutually exclusive) for better access and checking without too much knowledge about internals.